resolwe/flow/models/base.py

```diff
@@ -106,6 +110,6 @@ def save(self, *args, **kwargs):
 
                 raise
         else:
-            raise IntegrityError(
+            raise UniqueSlugError(
```

Comment:
Perhaps error could print what the max number of retries is?

Reply:
Sure, that will add some information.